src/app/shared/footer/footer.component.html

```diff
@@ -0,0 +1,31 @@
+<section class="footer-section">
+  <ng-container *ngIf="getStartedSectionVisible">
+    <div class="getStartedSection">
+      <div>
+        <h2 class="text firstLine">Ready to make a choice?</h2>
```

Comment:
It's better that a lower heading could be used because:
- H1 for the title at the top of the page 
- H2 for subsection